articles/tutorials/data-scenarios/index.md

```diff
@@ -0,0 +1,257 @@
+- [Why not put all the app’s data in the Auth0 data store?](/tutorials/user-data-storage-scenario#why-not-put-all-the-app-s-data-in-the-auth0-data-store-)
+- [When should I use the Auth0 data store?](/tutorials/user-data-storage-scenario#when-should-i-use-the-auth0-data-store-)
+- [Review](/tutorials/user-data-storage-scenario#review)
+
+
+## Where do I put my authentication data?
+
+Auth0 has a data store which serves as a way for developers to store data associated with their users’ profiles, beyond the basic information Auth0 uses for authentication, i.e name, email, username, password, and so on. You can also [Authenticate Users with Username and Password using a Custom Database](/connections/database/mysql) to store such information if your needs are different from what the Auth0 data store offers. However, we recommend storing authentication-related data in Auth0’s data store in order to make things less complicated by allowing you to easily manage user data through Auth0’s [dashboard](${uiURL}).
+
+
+## Why not put all the app’s data in the Auth0 data store?
+
+The Auth0 data store is highly specialized for storing authentication data. Storing any authentication-related data beyond the default user information in the Auth0 data store should only be done in specific cases. Here are some reasons why you should not use the Auth0 data store when you don't have to:
+
+### Scalability
+
+It is best not to store non-authentication-related data in the Auth0 data store because the Auth0 data store has limited scalability and your app’s data could exceed that limit. Using an external database allows you to keep things simple on the Auth0 side, to be able to efficiently manage your user authentication, while leaving the heavy database lifting up to a separate database formatted to efficiently store your extra data. 
+
+### Performance
+
+Beyond scale, keeping your authentication data separate from any other data in your app is important because the two sets of data are also likely accessed with different frequencies. The Auth0 data store is not optimized to be queried with extremely high frequency. It is better to leave this up to a specialized database service, optimized for fast, large data operations.
+
+### Flexibility 
+
+By using a separate database, your access to your users' authentication data is concentrated in a small chunk of code, and your access to all other data is grouped separately. Your organizational demands may not be met by the Auth0 data store because it is built to accomodate only the user profiles and their Metadata. Certain actions that you might require from a customizable database service are not possible in Auth0 data store. For example, if you wanted to make a specific query like `SELECT users.favGenre, access.roles FROM users, access WHERE users.user_id = access.user_id`. This is something you can't do with in the auth0 data store. Using a separate database will allow you to manage your data however you see fit.
+
```

Comment:
@eharkins I would replace these paragraphs (Scalability, Performance, Flexibility) with bullets, and make some redactions at the text as well. For example:

- __Scalability__: The Auth0 data store has limited scalability and your app’s data could exceed that limit. Using an external database allows you to keep things simple on the Auth0 side while leaving the heavy database lifting up to a separate database formatted to efficiently store your extra data.
- __Performance__: ...
- __Flexibility__: ...